HCore-Storage/Helpers/AwsHelpers.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using Amazon;
+using Amazon.S3;
+
+namespace HCore.Storage.Helpers
+{
+    public static class AwsHelpers
+    {
+        public const string AccessKeyIdKey = "AccessKeyId";
+        public const string SecretAccessKeyKey = "SecretAccessKey";
+
+        public const string DefaultRegion = "eu-central-1";
+
+        public static IAmazonS3 GetAmazonS3(IDictionary<string, string> connectionInfoByKey)
+        {
+            ArgumentNullException.ThrowIfNull(connectionInfoByKey);
+
+            if (!connectionInfoByKey.TryGetValue(AccessKeyIdKey, out var accessKeyId))
+            {
+                throw new Exception("Missing access key");
+            }
+
+            if (!connectionInfoByKey.TryGetValue(SecretAccessKeyKey, out var secretAccessKey))
+            {
+                throw new Exception("Missing secret access key");
+            }
+
+            if (!connectionInfoByKey.TryGetValue("Region", out var region))
+            {
+                region = DefaultRegion;
+            }
+
+            var regionEndpoint = RegionEndpoint.GetBySystemName(region);
+
+            return new AmazonS3Client(accessKeyId, secretAccessKey, regionEndpoint);
```

Comment:
Is this an object that should be created again and again, or is it reusable?

Reply:
It seems that it can be created again and again and can be reused

Do you want me to implement a hybrid approach where we use a shared AmazonS3Client for reading only and a new instance on upload?

Reply:
The question is: why is the memory increasing?

Reply:
I created dotnet trace for the migration tool in release mode using shared AmazonS3Client
This is what I get for uploading 996 assets
The memory usage isn't that high

<img width="2554" height="1390" alt="Image" src="https://github.com/user-attachments/assets/639fb913-6495-4843-93f6-25548d881cc7" />

Initially, when I was testing, I was observing a very slight increase in memory after each subsequent upload, but I cannot say why that is